README.md

```diff
@@ -1,43 +1,28 @@
 # sample-external-issuer
 
-This is an example of an [External Issuer] for cert-manager.
+External issuers extend [cert-manager](https://cert-manager.io/) to issue certificates using new APIs and services
```

Comment:
if I were to nitpick here, the word new doesn't seem right.
```suggestion
External issuers extend [cert-manager](https://cert-manager.io/) to issue certificates using APIs and services
```

Reply:
Post review comment: The suggested diff looks dumb because the sentence continues on the next line

Reply:
yeah, I think you're right, it has a feeling of "brand new" rather than "new to cert-manager". replaced :+1: